src/lib/main.js

```diff
@@ -175,4 +178,23 @@ module.exports = async (argv, env) => {
   }
 };
 
-const executeAction = action => require(`../fn/${action}`)();
+
+const requiresInstance = actions => actions.find(actionName => {
+  const actionLib = require(`../fn/${actionName}`);
```

Comment:
We also `require()` each action file in the `executeAction` function. Can we refactor so the action files are only required once?

Reply:
Well so the good news is that Node does cache these calls so they aren't actually doing anything too inefficient, but I agree it's a bit pants.

Can we address this in a later refactor? I didn't want to get too sidetracked, but alongside this there is quite a bit of cleanup to do around actions (their should be a module that deals with them instead of the code being dumped at the bottom of main, we shouldn't pass as couchUrl to actions that don't need it, etc)

Reply:
> we shouldn't pass as couchUrl to actions that don't need it, etc

Note that we don't pass a couchUrl to actions anymore